settings.gradle

```diff
@@ -0,0 +1 @@
+if (JavaVersion.current() >= JavaVersion.VERSION_1_8) include ':java8'
```

Comment:
I can't see where the name of the module is set - is this part of it? I'd be keen for it to be named concordion-junit5 rather than concordion-java8.